content/rs/administering/cluster-operations/updating-certificates.md

```diff
@@ -26,6 +26,26 @@ future sessions.{{% /note %}}
 
 {{% warning %}}The new certificate replaces the equivalent certificate on all nodes in the cluster. Existing certificates are overwritten.{{% /warning %}}
 
+- For Redis Enterprise version 5.4.0 and above use rladmin
+
+    ```bash
+
+    rladmin> cluster certificate set <cert_name> certificate_file <cert> key_file <key>
```

Comment:
@RoeyPrat Can you verify that this addition is correct?

Reply:
certificate_file arg is the path to the certificate, not the "text" of it. 
this might be clear as-is, i'm not sure.

In comparison, the rladmin docs say:
`cluster certificate set <certificate_name> certificate_file <certificate_file>
    [key_file <certificate_file>]`